app/storage.py

```diff
@@ -115,19 +115,20 @@ def get_scans(path='storage/scans/', withDetails=False):
     scans = sorted(scans, key=lambda x: x['creation_date'], reverse=True)
 
     scans_with_status = []
-    for s in scans:
-        
+    for s in scans:   
         if os.path.exists(os.path.join(s['path'],'sunscan_clahe.jpg')):
             s['status'] = 'completed'
         elif os.path.exists(os.path.join(s['path'],'sunscan_log.txt')):
             s['status'] = 'failed'
 
         if os.path.exists(os.path.join(s['path'], 'sunscan_conf.txt')):
-            d = open(os.path.join(s['path'], 'sunscan_conf.txt'))
-            c = json.load(d)
-            s['configuration'] = c
+            try:
+                d = open(os.path.join(s['path'], 'sunscan_conf.txt'))
+                c = json.load(d)
+                s['configuration'] = c
+            except:
+                pass
```

Comment:
Tu peux écrire ça plus simplement avec un with, c'est préférable car ça évite les fichier ouvert indéfiniment notament. Et ça remplace aussi ton `if os.path.exists(os.path.join(s['path'], 'sunscan_conf.txt')):`
```suggestion
        with open(os.path.join(s['path'], 'sunscan_conf.txt')) as d:
            c = json.load(d)
            s['configuration'] = c
        except FileNotFoundError:
            # Le fichier n'existe pas, on passe simplement
            pass 
        except Exception as e:
            # Vous pouvez gérer d'autres erreurs ici (par exemple, des erreurs de parsing JSON)
            pass